doc/src/build-instructions.md

```diff
@@ -315,6 +315,63 @@ $ pytest --embedded-services=idf,qemu -s
 
 ESP32 tests are erlang modules located in `src/platforms/esp32/test/main/test_erl_sources/` and executed from `src/platforms/esp32/test/main/test_main.c`.
 
+### Performance and Power
+
+AtomVM comes with conservative defaults for broad compatibility with different ESP32 boards, and a reasonable performance/power/longevity tradeoff.
+
+You may want to change these settings to optimize for your specific applications performance and power needs.
+
+Factors like heat dissipation should also be considered, and the effect on overall longevity of components.
+
+#### CPU frequency
+
+Use `idf.py menuconfig` in `src/platforms/esp32`
+```Component config ---> ESP System Settings  ---> CPU frequency (160 MHz)  --->```
```

Comment:
Sphinx is picky about the triple backticks, they should be on a separate line by themselves (unless they are a style tag like "```shell" for example)... if you want this in a separate code block you should use
```
Component config ---> ESP...
```
Otherwise if it fits on a single like you can surround it with single backticks, and it will be inline monospaced code quote style, that is up to you.